azkaban-webserver/src/main/less/flow.less

```diff
@@ -340,3 +340,17 @@ li.tree-list-item {
     }
   }
 }
+
+#analyzerButton {
```

Comment:
The simplest way to put this button without any CSS change was the below.

<li class="nav-button pull-right"><form target="_blank"><button title="Analyze job in Dr. Elepahnt" formaction="http://google.com/search?q=banana" class="btn btn-info btn-sm" id="analyzeLink" type="link">Dr. Elephant</button></form></li>

Note the use of 'formaction' which is supported in HTML5. However, there's a bug in Chromium and webkit
whenever there is a '?' in the url. This is explained in the following blog.

http://tanalin.com/en/blog/2013/03/link-button/

Hence, I have chosen an anchor to represent this and attribute type="button" will cause bootstrap to make
it look like a button. Due to other overrides I had to define it on the ID.

Alternately, I can write it as follows which will be using existing classes.

.nav.nav-sm > li > a[type="button"] {
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 3px;
  
    &:hover, &:focus {
      color: #ffffff;
      text-decoration: none;
	  border-color: #39b3d7;
	  background-color: #269abc;        
    }    
}

I hope this is fine?